src/Lorentz/Contracts/TZBTC/V1/ManagedLedger/Types.hs

```diff
@@ -15,6 +15,16 @@ module Lorentz.Contracts.TZBTC.V1.ManagedLedger.Types
     ) where
 
 import Lorentz
+    ( Bool,
+      Natural,
+      Map,
+      type (:!),
+      StorageContains,
+      type (~>),
+      Dupable,
+      type (:=),
+      Address,
+      View )
```

Comment:
```suggestion
import Lorentz
  ( Address, Bool, Dupable, Map, Natural, StorageContains, View, type (:!)
  , type (~>), type (:=)
  )
```
:nabros:

Reply:
I'm pretty sure that this change was added unintentionally by the haskell VS Code plugin